.gitignore

```
@@ -3,4 +3,9 @@ target/
 Cargo.lock
 .DS_Store
 .cargo
+*.png
 
+## Intellij Idea project configuration files
+.idea/
+## VScode project configuration files
+.vscode/
```

we might want to add the newline here or some things will complain
```suggestion
.vscode/

```